src/contact.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: default.html
+---
+
+<h2>contact us</h2>
+<p>Contact us for any questions about catering, the events we’re at, biscuits in general or just to say hello. Use the form on the right or contact us directly via: 
```

Comment:
Avoid the use of `br` elements here and remember to make the email address a usable link.